### ISSUE TITLE
"Up-arrow" in editing field should not change focus

### ISSUE THREAD

**Issue by holmesworcester** [TryQuiet/quiet#669]
I thought UpArrow was an acceptable substitute for PageUp when switching out of the editing field to scroll up the message list, using the keyboard, but it's not, since it's used so often for editing while composing a message and this change focus behavior would not be expected by the user. 

Solution: we should use pageup for this or, if that's difficult, require the user to use shift-tab (which works currently)

**Comment by pdurbin** [comment]
Related:

- #1339

**Comment by pdurbin** [comment]
As of 1.2.0 this seems to be fixed for me. Close it?

**Comment by kingalg** [comment]
Version: 1.2.0-alpha.30

Support for moving in message field with arrows is not yet there but we are not changing focus by moving arrows and the rest should be resolved in issue #1339 so I'm closing this one.
